### PR TITLE
course dependencies data: Build a Developer Blog with Gatsby

### PR DESCRIPTION
**Pull request by laurosilvacom**
- [x]  Add freshness review
- [x]  Add dependencies
    - [x]  topics
    - [x]  questions to reflect upon (essential)
- [x]  Associate related resources
- [x]  Freshness review for old Gatsby course
- [x]  Associate related resources to old Gatsby course
- [x]  Add a custom open graph image

image

image

### CODE REVIEW

File: src/data/courseDependenciesData.js [hunk 1]
```diff
@@ -3,6 +3,121 @@ import {find} from 'lodash'
...
+        ],
+      },
+      {
+        id: '157',
+        type: 'course',
+        slug: 'build-a-blog-with-react-and-markdown-using-gatsby',
+        dependencies: {
+          gatsby: '^2.0.0',
+        },
+        freshness: {
```
Comment (laurosilvacom) [inline src/data/courseDependenciesData.js:80]: Freshness review to old Gatsby course, [Build a Blog with React and Markdown using Gatsby](https://egghead.io/courses/build-a-blog-with-react-and-markdown-using-gatsby)

**Comment by zacjones93** [comment]
love the share card 
image

File: src/data/courseDependenciesData.js [hunk 2]
```diff
@@ -3,6 +3,123 @@ import {find} from 'lodash'
...
+            slug:
+              'building-a-serverless-jamstack-todo-app-with-netlify-gatsby-graphql-and-faunadb-53bb',
+            description: `This application serves as an introduction to building products with a JAMStack and Serverless approach.`,
+            image:
+              'https://d2eip9sf3oo6c2.cloudfront.net/tags/images/000/001/211/full/gatsby.png',
+          },
+        ],
+      },
+      {
+        id: '157',
```
Comment (zacjones93) [inline src/data/courseDependenciesData.js:76]:
```suggestion
        id: 432685,
```

File: src/data/courseDependenciesData.js [hunk 3]
```diff
@@ -3,6 +3,128 @@ import {find} from 'lodash'
...
+        slug: 'build-a-developer-blog-with-gatsby-bd96',
+        illustrator: {
+          name: 'Maxime Bourgeois',
+        },
+        customOgImage:
+          'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1614666866/next.egghead.io/cards/build-a-developer-blog-with-gatsby_2x.png',
+        dependencies: {
+          gatsby: '>=2.27.1',
+        },
+        courseProject: {
```
Comment (laurosilvacom) [inline src/data/courseDependenciesData.js:19]: Links to portfolio challenge.

Comment (zacjones93) [reply src/data/courseDependenciesData.js:19]: referencing #487 for posterity

**Review by zacjones93 — APPROVED** [review]

**Pull request merged**